styles.css

```diff
@@ -0,0 +1,108 @@
+* {
+margin: 0;
+padding: 0;
+box-sizing: border-box;
+}
+
+.app {
+    width: 100vw;
+    min-height: 100vh;
+    background-image: url("images/background.png");
+    background-repeat: no-repeat;
+    background-size: cover;
+
+}
+
+
+.nav-bar-list {
+    display: flex;
+    gap: 32px;
+    align-items: center;
+}
+
+.nav-bar-list-right {
+    display: flex;
+    gap: 12px;
+    align-items: center;
+}
+
+
+.nav-link {
+    list-style: none;
+}
+
+
+.nav-link-text {
+    font-family: 'inter', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    color: #18191f;
+    text-decoration: none;
+    
+}
+
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    justify-content: space-between;
+    margin: auto;
+}
+
+
+.hero-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 1440px;
+    margin: 100px auto auto;
+    padding: 0 20px;
+    
+}
+
+.hero-title {
+    font-family: 'Inter', sans-serif;
```

Comment:
- [ ] `font-family` property'sini genelde *{} içersinide ya da body tag in içerisine yazman daha iyi olur böylece tekrardan her div içerisinde belirtmene gerek kalmaz